nrf_rpc/CHANGELOG.rst

```diff
@@ -0,0 +1,106 @@
+.. _nrf_rpc_changelog:
+
+Changelog
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+All notable changes to this project in the |NCS| are documented in this file.
+
+Main branch
+***********
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+
+* Added configurations number of ZCBOR backups.
+
+nRF Connect SDK v2.1.0
+**********************
+
+All the notable changes added to the |NCS| v2.1.0 release are documented in this section.
+
+Added
+=====
+
+* Added support for multi-instance transport.
+  Each group can use a different transport instance, or groups can share a single transport instance.
+* Added support for the nRF RPC protocol versioning.
+
+Changes
+=======
+
+* Updated documentation.
+
+nRF Connect SDK v2.0.0
+**********************
+
+All the notable changes added to the |NCS| v2.0.0 release are documented in this section.
+
+Changes
+=======
+
+* Updated figures to follow style guidelines.
+* Moved from TinyCBOR to zcbor.
+
+nRF Connect SDK v1.9.0
+**********************
+
+No changes in this release.
+
+nRF Connect SDK v1.8.0
+**********************
+
+All the notable changes added to the |NCS| v1.8.0 release are documented in this section.
+
+Changes
+=======
+
+* Removed Latin abbreviations from the documentation.
+
+nRF Connect SDK v1.7.0
+**********************
+
+All the notable changes added to the |NCS| v1.7.0 release are documented in this section.
+
+Changes
+=======
+
+* Used :kconfig: role as a reference to the Kconfig options.
```

Comment:
We use :kconfig:option:`CONFIG_OPTION_NAME` for the formatting, but I could not find any instances within the RPC docs in nrfxlib.

Reply:
That's right, this change relates to the template header file.